classes/Cart.php

```diff
@@ -1019,7 +1019,7 @@ public function updateQty(
                 } else {
                     Db::getInstance()->execute(
                         'UPDATE `'._DB_PREFIX_.'cart_product`
-                        SET `quantity` = `quantity` '.$qty.', `date_add` = NOW()
+                        SET `quantity` = `quantity` '.$qty.'
```

Comment:
this can't work (I know it was here before you @maximebiloe  ^^  )

```
mysql> show columns from ps_cart_product;
+----------------------+------------------+------+-----+---------+-------+
| Field                | Type             | Null | Key | Default | Extra |
+----------------------+------------------+------+-----+---------+-------+
| id_cart              | int(10) unsigned | NO   | PRI | NULL    |       |
| id_product           | int(10) unsigned | NO   | PRI | NULL    |       |
| id_address_delivery  | int(10) unsigned | NO   | PRI | 0       |       |
| id_shop              | int(10) unsigned | NO   |     | 1       |       |
| id_product_attribute | int(10) unsigned | NO   | PRI | NULL    |       |
| quantity             | int(10) unsigned | NO   |     | 0       |       |
| date_add             | datetime         | NO   |     | NULL    |       |
+----------------------+------------------+------+-----+---------+-------+
```

what do you think ?